assets/helpers/contributions.js

```diff
@@ -21,43 +26,45 @@ export type Amounts = {
   oneOff: Amount,
 };
 
+export type validatedContrib = {
+  amount: number,
+  error: ContribError,
+};
+
 
 // ----- Setup ----- //
 
-const limits = {
+const LIMITS = {
   max: 2000,
   min: {
-    monthly: 5,
-    oneOff: 1,
+    RECURRING: 5,
+    ONE_OFF: 1,
   },
 };
 
+const DEFAULT_AMOUNTS = {
+  RECURRING: 10,
+  ONE_OFF: 50,
+};
 
-// ----- Functions ----- //
 
-function validate(amount: Amount, contrib: Contrib): ?ContribError {
+// ----- Functions ----- //
 
-  if (amount.value === '') {
-    return 'invalidEntry';
-  }
+export default function validate(input: string, contrib: Contrib) {
```

Comment:
Whoops, I [created](https://github.com/guardian/support-frontend/pull/142/files/260c4d52ccaa40a32c8cfc91362cc05ed0605f00#diff-547379c25f3f083eba0df6432e19d764R29) a type for this and forgot to add it.